zone/client_packet.cpp

```diff
@@ -4889,13 +4889,19 @@ void Client::Handle_OP_ConsiderCorpse(const EQApplicationPacket *app)
 		);
 
 		if (target->IsPlayerCorpse()) {
-			Message(
+			uint32 rez_time = target->GetRemainingRezTime();
+			if (rez_time > 0) {
+				auto rez_time_string = Strings::SecondsToTime(rez_time, true);
+				Message(
 				Chat::NPCQuestSay,
 				fmt::format(
-					"This corpse {} be resurrected.",
-					target->IsRezzed() ? "cannot" : "can"
+					"This corpse's resurrection time will expire in {}",
+					rez_time_string
 				).c_str()
 			);
+			} else {
+				Message(Chat::NPCQuestSay, "This corpse is too old to be resurrected.");
```

Comment:
We should be able to switch this and the ressurection time message for eqstrs. Would have to confirm with live but probably these
```
231 This player cannot be resurrected. The corpse is too old.
491 This corpse's resurrection time will expire in %1 hour(s) %2 minute(s) %3 seconds.
492 This corpse's resurrection time will expire in %1 minute(s) %2 seconds.
```